.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,56 @@
+---
+name: New Release
+about: Propose a new release
+title: Release vX.y.z
+
+---
+
+## Release Checklist
+<!--
+Please do not remove items from the checklist
+-->
+- [ ] Network-operator related component versions in Helm chart are up to date. (otherwise, submit PR to update)
+  - [ ] node-feature-discovery
+  - [ ] SR-IOV Network Operator
+  - [ ] Manifest related component default versions
+- [ ] Example folder is up to date (otherwise submit PR to update)
+- [ ] Update network-operator Helm `Chart.yaml` with the release version (`appVersion`, `version` fields)
+  - [ ] Ensure Helm CI is passing on updated Chart.
+- [ ] Tag release
+- [ ] Create release in github
+  - [ ] Changelog for current release with: `New Features`, `Bug Fixes`, `Known Limitations`
```

Comment:
where is the changelog is captured?

Reply:
it is captured in two places
1. in the release issue
2. in the github release itself (when you draft a release, you can fill info about the release.)

The description of the release will contain 3 sections : New features, bug fixes, known limitations.
i will explain it better here.